platform/viewer/src/routes/DataSourceWrapper.tsx

```diff
@@ -32,11 +32,16 @@ function DataSourceWrapper(props) {
     return acc.concat(mods);
   }, []);
 
-  // Grabbing first for now - should get active?
-  const name = webApiDataSources[0].name;
+  // Grabbing first defined for now - should get active
   // TODO: Why does this return an array?
-  const dataSource = extensionManager.getDataSources(name)[0];
-
+  const dataSource = webApiDataSources
```

Comment:
The logic is now to find the first defined data source - that isn't technically correct, as there should be a way to set it from configuration, but I was mostly just trying to get the base definitions working with an extension.  @sedghi  - if you want to use configuration, if you could suggest how to do that, I'd appreciate it.